pretix_sepadebit/payment.py

```diff
@@ -417,9 +417,11 @@ def render_invoice_text(self, order, payment: OrderPayment) -> str:
         ref = '%s-%s' % (self.event.slug.upper(), order.code)
         if self.settings.reference_prefix:
             ref = self.settings.reference_prefix + "-" + ref
-        t = _("We will debit the total amount of this order from your bank account by "
+        iban_redacted = 'XXXX ' + payment.info_data['iban'][-4:]
```

Comment:
I could not find a „standard“ as to how to anonymize/redact ibans. Is the country-code and the first two digits enough? I guess country-code can easily be guessed based on the buyers address … I found [this](https://lbv.landbw.de/-/teilweise-anonymisierung-der-kontoverbindung-auf-lbv-schreiben-und-bescheiden) (german) where they show the first 3 and the last 3 digits/chars and replace the rest in the middle.

Reply:
I don't think there is a standard for it

Reply:
Note that this is showing the last four digits, not everything except the first 4 as your commend suggests